app/src/main/res/layout/activity_page.xml

```diff
@@ -13,6 +13,14 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
+        <ImageView
```

Comment:
Couldn't the image be loaded directly into the `PageHeaderView`? (and no longer require a cached bitmap to be kept in memory?)

Reply:
Do you mean only the image or the image with the article summary? I have tried to implement the animation for only the image without any cached bitmap, just like what we did for the floating queue thing, I found that the animation is not as much smooth as the way we did before.

I guess that's because of the changes after we implemented `mobile-html`.

Also, looks like Robin wants the image+text animation: https://phabricator.wikimedia.org/T263259#6503265